frontend/ui/widget/dictquicklookup.lua

```diff
@@ -438,19 +433,13 @@ function DictQuickLookup:init()
                 },
                 {
                     id = "highlight",
-                    text = self:getHighlightText(),
+                    text = _("Highlight"),
```

Comment:
Yes, this feature is removed. Do you think it should be restored?

Reply:
Dunno. I don't use it, so can't say. I just gave a fictional use case :) Needs other people opinions.

(I think that may be, unrelated to this PR, this <kbd>Highlight</kbd> button could be replaced by a <kbd>☰|☰</kbd> allowing to dismiss the DictQuickLookup window and get the multiple-word-highlight-dialog to do other things from there - could be more useful than this <kbd>Highlight</kbd> - and may be the <kbd>Unhighlight</kbd> still works on that ButtonTable?.)

Reply:
> this Highlight button could be replaced by a ☰|☰

Agree, it would be a good consistent button. I'll try to add in a separate PR after finalizing this one.